models/classes/interface.ReadableResultStorage.php

```diff
@@ -82,6 +82,12 @@
      */
     public function getVariables($callId);
 
+    /**
+     * Get the complete variables list stored for delivery execution
+     * @param $deliveryResultIdentifier
+     * @return mixed
+     */
+    public function getDeliveryVariables($deliveryResultIdentifier);
```

Comment:
please put this method in ResultManagement interface
We try to not introduce method in this interface